oecophylla/cli/launch.py

```diff
@@ -86,67 +97,118 @@ def _create_dir(_path):
               help='Restarts the run and overwrites previous input.')
 @click.option('--just-config', is_flag=True, flag_value=False,
               help='Only generate the configuration file.')
+@click.option('--test', is_flag=True, default=False,
+              help='Executes a run with the included test data.')
 def workflow(targets, input_dir, sample_sheet, params, envs,
```

Comment:
Can you update the Readme on how to test run the workflow?

Reply:
Sure thing.